alembic.ini

```diff
@@ -0,0 +1,114 @@
+# A generic, single database configuration.
```

Comment:
I know absolutely nothing about Alembic and an attempt to skim this stuff to see if I could add any value through a brief review failed :) so I'm skipping all the Alembic stuff for now.  Let me know if there's anything here I should pay particular attention to. Else at some point I'll try to read up on it :D

Reply:
Nothing under `\alembic` is worth reading, as it's all autogenerated. Will add overview of what it is and links to documentation. Goal of it is to never need to manually do database migrations, and instead be able to simply run a script that given the changes to SQLAlchemy models in the code is able to migrate the database safely